### DISCUSSION TITLE
Does this work with C# projects

### DISCUSSION THREAD

**Discussion by Oreboat** [dialogic-godot/dialogic#1631] [Q&A]
I loved using the addon in 3.X and just discovered it also exists for 4.0 but I was wondering if this works for C# since I've been using it with Godot 4

**Comment by Jowan-Spooner** [comment]
Hey @Oreboat, 
unfortunately I have absolutely NO idea. I'm not sure why it shouldn't work like any other gdscript class you access from C#. 
That said, I personally don't (and as far as I know noone on the core-team does) use C# in godot, so it has never been a priority. 
If you can test it however and see if it works, that would be very valuable information.

**Comment by QueenOfSquiggles** [comment]
Hey there! I wrote some code to interface with Dialogic in Godot 4.1.1 with C#. It's incredibly simple but I'm also planning on building a C# wrapper (discussion here #1776 ). If you like, feel free to just rip the interfacing code I wrote for my ludum dare project [here](https://github.com/QueenOfSquiggles/ContactRange/blob/main/Game/NPC/States/StatePlayTrack.cs).

Since it was for the jam I didn't really add a lot of comments so if anything is confusing feel free to ask! I hope this helps with what you're wanting to do!! :smile:
